client/components/select-dropdown/style.scss

```diff
@@ -48,6 +48,7 @@
 	font-size: 14px;
 	font-weight: 600;
 	line-height: 18px;
+	height: 18px;
```

Comment:
I've set this height because before to get the timezones response the dropdown is empty it produces a visual bug like the the following:

![image](https://cloud.githubusercontent.com/assets/77539/14090594/afaa79e8-f511-11e5-8630-a6c5d2ed8581.png)